src/integration-test/java/com/commercetools/sync/integration/sdk2/commons/utils/TaxCategoryITUtils.java

```diff
@@ -0,0 +1,126 @@
+  public static final String TAXCATEGORY_DESCRIPTION_1 = "description_1";
+
+  /**
+   * Deletes all Tax categories from CTP projects defined by the {@code CTP_SOURCE_CLIENT} and
+   * {@code CTP_TARGET_CLIENT}.
+   */
+  public static void deleteTaxCategoriesFromTargetAndSource() {
+    deleteTaxCategories(CTP_TARGET_CLIENT);
+    deleteTaxCategories(CTP_SOURCE_CLIENT);
+  }
+
+  /**
+   * Deletes all tax categories from the CTP project defined by the {@code ctpClient}.
+   *
+   * @param ctpClient defines the CTP project to delete the tax categories from.
+   */
+  public static void deleteTaxCategories(@Nonnull final ProjectApiRoot ctpClient) {
+    final Consumer<List<TaxCategory>> taxCategoryConsumer =
+        taxCategories -> {
+          CompletableFuture.allOf(
+                  taxCategories.stream()
+                      .map(taxCategory -> deleteTaxCategoryWithRetry(ctpClient, taxCategory))
+                      .map(CompletionStage::toCompletableFuture)
+                      .toArray(CompletableFuture[]::new))
+              .join();
+        };
+    QueryUtils.queryAll(ctpClient.taxCategories().get(), taxCategoryConsumer)
+        .handle(
+            (result, throwable) -> {
+              if (throwable != null && !(throwable instanceof NotFoundException)) {
+                return throwable;
+              }
+              return result;
+            })
+        .toCompletableFuture()
+        .join();
+  }
+
+  private static CompletionStage<TaxCategory> deleteTaxCategoryWithRetry(
+      @Nonnull final ProjectApiRoot ctpClient, @Nonnull final TaxCategory taxCategory) {
+    return ctpClient
+        .taxCategories()
+        .delete(taxCategory)
+        .execute()
+        .thenApply(ApiHttpResponse::getBody);
+  }
+
+  /**
+   * Creates a {@link TaxCategory} in the CTP project defined by the {@code ctpClient} in a blocking
+   * fashion. The created tax category will have a key with the value {@value TAXCATEGORY_KEY}, a
+   * name with the value {@value TAXCATEGORY_NAME}, a description with the value {@value
+   * TAXCATEGORY_DESCRIPTION} and a tax rate with the name {@value TAXCATEGORY_TAXRATE_NAME} and
+   * amount {@value TAXCATEGORY_TAXRATE_AMOUNT}.
+   *
+   * @param ctpClient defines the CTP project to create the tax category in.
+   * @return the created tax category.
+   */
+  public static TaxCategory createTaxCategory(@Nonnull final ProjectApiRoot ctpClient) {
+    final TaxCategoryDraft taxCategoryDraft =
+        TaxCategoryDraftBuilder.of()
+            .name(TAXCATEGORY_NAME)
+            .rates(createTaxRateDraft())
+            .description(TAXCATEGORY_DESCRIPTION)
+            .key(TAXCATEGORY_KEY)
+            .build();
+
+    return ctpClient
+        .taxCategories()
+        .post(taxCategoryDraft)
+        .execute()
+        .thenApply(ApiHttpResponse::getBody)
+        .join();
+  }
+
+  /**
+   * Creates a {@link io.sphere.sdk.taxcategories.TaxRateDraft} with the name {@value
```

Comment:
Replace the link with path to the v2 model.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/920/commits/8f1cfc505581c5b334ea7f95faeb1a529331feff